tests/frontend/ui/test_home.py

```diff
@@ -92,13 +92,13 @@ def test_add_ons_footer_links(base_url, selenium, i, links):
     enumerate([
         'firefox/new',
         'firefox/mobile',
-        'firefox/mobile',
+        'mixedreality.mozilla.org',
         'firefox',
         'firefox/channel/desktop',
     ])
 )
 @pytest.mark.nondestructive
-def test_firefox_footer_links(base_url, selenium, i, links):
+def test_browsers_footer_links(base_url, selenium, i, links):
     page = Home(selenium, base_url).open()
-    page.footer.firefox_links[i].click()
+    page.footer.browsers_links[i].click()
```

Comment:
I don't see this ```browser_links``` property within base class. You didn't change the name in your last PR. https://github.com/mozilla/addons-frontend/blob/master/tests/frontend/ui/pages/desktop/base.py#L185

Reply:
Done now